### PR TITLE
[15.0][ADD] website_sale_stock_provisioning_time

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: website_sale_stock_provisioning_time/views/website_sale_product.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    id="product_website_sale_privious"
+    name="product_sale"
+    inherit_id="website_sale.product"
+  >
+        <xpath expr="//a[@id='add_to_cart']" position="after">
+            <div class="add_atribute" style="margin-top:10px;">
+                <t t-if="product.provisioning_time_in_workdays">
+                    <span>Provisioning time until shipment:</span>
+                    <span t-esc="product.provisioning_time_in_workdays" />
+                    Work days
```
Comment (alihashaam) [inline website_sale_stock_provisioning_time/views/website_sale_product.xml]: if the field has value 1 for the product, the output will be "1 Work days" and that is grammatically not right.

**Comment by alihashaam** [comment]
@dsolanki-initos 

I have requested one small change in this PR. So can you please do that

File: website_sale_stock_provisioning_time/views/website_sale_product.xml (hunk 1 above)
Comment (dsolanki-initos) [reply website_sale_stock_provisioning_time/views/website_sale_product.xml]: I'll fix it

**Comment by dsolanki-initos** [comment]
@alihashaam, I've done the changes, Please check it.

File: website_sale_stock_provisioning_time/views/website_sale_product.xml [hunk 2]
```diff
@@ -11,7 +11,12 @@
                 <t t-if="product.provisioning_time_in_workdays">
                     <span>Provisioning time until shipment:</span>
                     <span t-esc="product.provisioning_time_in_workdays" />
-                    Work days
+                    <t t-if="product.provisioning_time_in_workdays == 1">
+                        Work day
```
Comment (jans23) [inline website_sale_stock_provisioning_time/views/website_sale_product.xml:15]: It lacks translation. German: "Arbeitstag"

File: chatter_confirm_message/static/src/components/composer/composer.esm.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
+import Dialog from "web.Dialog";
 import {_t} from "web.core";
 import {registerMessagingComponent} from "@mail/utils/messaging_component";
-import {useDragVisibleDropZone} from "@mail/component_hooks/use_drag_visible_dropzone/use_drag_visible_dropzone";
-import Dialog from "web.Dialog";
 import rpc from "web.rpc";
+import {useDragVisibleDropZone} from "@mail/component_hooks/use_drag_visible_dropzone/use_drag_visible_dropzone";
 
 const {useRef} = owl.hooks;
 const {Component} = owl;
```
Comment (alihashaam) [inline chatter_confirm_message/static/src/components/composer/composer.esm.js:11]: @dsolanki-initos 

What is this change about?

File: website_sale_stock_provisioning_time/views/website_sale_product.xml (hunk 2 above)
Comment (alihashaam) [reply website_sale_stock_provisioning_time/views/website_sale_product.xml:15]: I have added the translation

**Comment by alihashaam** [comment]
@jans23 this PR is ready to merge

**Pull request merged**